mods/common/languages/en.ftl

```diff
@@ -172,5 +283,289 @@ none = None
 ## LobbyLogic, IngameChatLogic
 team = Team
 
-## ServerListLogic, ReplayBrowserLogic also ObserverShroudSelectorLogic
+## LobbyOptionsLogic
+not-available = Not Available
+
+## LobbyUtils
+slot = Slot
+open = Open
+closed = Closed
+bots = Bots
+
+# LobbyUtils, Server
+bots-disabled = Bots Disabled
+
+## MapPreviewLogic
+connecting = Connecting...
+downloading-map = Downloading { $size } kB
+downloading-map-progress = Downloading { $size } kB ({ $progress }%)
+retry-install = Retry Install
+retry-search = Retry Search
+## also MapChooserLogic
+created-by = Created by { $author }
+
+## SpawnSelectorTooltipLogic
+disabled-spawn = Disabled spawn
+available-spawn = Available spawn
+
+## DisplaySettingsLogic
+close = Close
+medium = Medium
+far = Far
+furthest = Furthest
+
+windowed = Windowed
+legacy-fullscreen = Fullscreen (Legacy)
+fullscreen = Fullscreen
+
+display = Display { $number }
+
+show-on-damage = Show On Damage
+always-show = Always Show
+
+automatic = Automatic
+manual = Manual
+
+## DisplaySettingsLogic, InputSettingsLogic
+disabled = Disabled
+
+## DisplaySettingsLogic, InputSettingsLogic, IntroductionPromptLogic
+classic = Classic
+modern = Modern
+standard = Standard
+
+## DisplaySettingsLogic, IntroductionPromptLogic
+inverted = Inverted
+joystick = Joystick
```

Comment:
Most of these identifiers are too vague or generic. I'd suggest adding more context to them. For example:
```ftl
camera-close = Close
camera-medium = Medium
camera-far = Far
camera-furthest = Furthest

display-mode-windowed = Windowed
display-mode-legacy-fullscreen = Fullscreen (Legacy)
display-mode-fullscreen = Fullscreen
video-display-index = Display { $number }
status-bars-show-on-damage = Show On Damage
status-bars-always-show = Always Show

target-lines-automatic = Automatic
target-lines-manual = Manual
target-lines-disabled = Disabled

control-scheme-classic = Classic
control-scheme-modern = Modern
control-scheme-standard = Standard

mouse-scroll-type-inverted = Inverted
mouse-scroll-type-joystick = Joystick

Reply:
Addressed in #20207

Reply:
I disagree, this allows easier re-use on other places.

Reply:
It actually makes reuse harder because a translation can't be used in different contexts. For example what does "Close" mean? Is it a verb (used in an dialog button "Close [the dialog]") or is it something else (a adjective in this case "Close [camera]"). Or "Classic" for example - is it plural, what gender is appropriate? That's why providing context for the translations is important.